### DISCUSSION TITLE
Cannot change brightness after setting to standard gpu mode

### DISCUSSION THREAD

**Discussion by Frigidmoon** [seerge/g-helper#886] [Q&A]
Flow X13 2023 (GV302XV) here.
Perhaps having to do with the recent ASUS System Control Interface v3 update (V3.1.16.0 on 2023/07/21), or something I did wrong in my lastest windows 11 reinstall (did a fresh install because swapped a ssd), but going from eco mode to standard mode from a eco mode boot seems to disable brightness adjustment. The slider is there, but does not do anything in standard mode. Is anyone else having this problem or something on my end? Armoury Crate seems to be fine though, and after changing to standard with armoury crate, Ghelper standard is able to change brightness as well

**Comment by joe-700** [comment]
This is related to the issue with Eco mode switching, the prevailing theory is that this is an issue with the BIOS that ASUS need to address and only occurs when the 2023 laptops shutdown in eco mode. The latest update; 0.101.0.0 addresses this by forcing the GPU switch to standard mode before shutdown. You will need to add a line to the config file for it to work, as per the power user instructions.

**Comment by rainbowflesh** [comment]
Simply re-enable nvi driver in `Device Manager` will help.
Flow like: change to `standard` lose brightness control -> keep in `standard` and *disable* nvi driver -> change to `echo`, this will make GHelper stay in `changing` statue -> enable driver -> GHelper will change to `echo`, and now back to `standard` will restore brightness control.

**Reply by haveagudweek** [reply]
any new solution? should i do it everytime i use standard?

**Reply by rainbowflesh** [reply]
> any new solution? should i do it everytime i use standard?

![QQ_1745405491199](https://github.com/user-attachments/assets/d76854a0-1622-4031-99f3-f00c4ec6d851)
